unskript-ctl/unskript_ctl_notification.py

```diff
@@ -294,7 +299,7 @@ def create_checks_summary_message(self,
             message += tr_message + '\n'
             message += '</table>' + '\n'
 
-
+        self.logger.info(message)
```

Comment:
why do we need this

Reply:
This will be addressed as https://unskript.atlassian.net/browse/EN-5240